library/common-library/templates/_labels.tpl

```diff
@@ -16,6 +16,12 @@ helm.sh/chart: {{ include "common.naming.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- with .Values.global.labels }}
+{{ toYaml . }}
+{{- end -}}
+{{- with .Values.labels }}
+{{ toYaml . }}
+{{- end -}}
```

Comment:
It wouldn't be better if we merge them "just in case"?

Reply:
Well, we would have to do more magic creating a dict with `app.kubernetes.io/version`, `app.kubernetes.io/managed-by`,... so we can merge that too.

Maybe it is a bit overkill. WDYT?

Reply:
- yes, I feel it would be a bit obscure to create a dict with `app.kubernetes.io/version, app.kubernetes.io/managed-by`. If needed anyone can still overriding them

 - regarding the merge I followed the approach of:
```
{{- if include "common.labels.overrides.addLabels" . }}
{{ include "common.labels.overrides.addLabels" . }}
{{- end }}
```
since it was simpler. We would need to merge 4 sources (standard, user, global, override) and it could be a bit confusing. But If you think it is better I am fine with changing it